pkgs/development/dotnet-modules/python-language-server/updater.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p gnused jq common-updater-scripts nuget-to-nix dotnet-sdk_3 nix-prefetch-git
+#!nix-shell -i bash -p gnused jq common-updater-scripts nix-prefetch-git
```

Comment:
This seems to be missing `-I nixpkgs=../../../..`?
```suggestion
#!nix-shell -I nixpkgs=../../../.. -i bash -p gnused jq common-updater-scripts nix-prefetch-git
```